### PR TITLE
Support undefined children

### PR DESCRIPTION
**Pull request by eiof**
See #18 for details

### CODE REVIEW

File: lib/ReactJsonSchema.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ export default class ReactJsonSchema {
 
   resolveComponentChildren(schema) {
     return (schema.hasOwnProperty('children')) ?
```
Comment (nlaffey) [inline lib/ReactJsonSchema.js]: Should this have a check to make sure that the property children is not null? I believe this throws an error if we pass this:

```{
component: input,
children: , 
}```

Comment (eiof) [reply lib/ReactJsonSchema.js]: That's not valid JavaScript or JSON. And null is an acceptable value for children. If there's other concerns, we can create a new issue around the subject.

Comment (nlaffey) [reply lib/ReactJsonSchema.js]: Sorry, I wrote that in a hurry. I meant to write:

```
{
'component': 'input',
'children' : null,
'value' : 'foo'
}
```

This throws an error: 

```
Uncaught TypeError: Cannot read property 'component' of null
    at ReactJsonSchema.createComponent (bundle.js:49558)
    at ReactJsonSchema.parseSchema (bundle.js:49515)
    at ReactJsonSchema.resolveComponentChildren (bundle.js:49588)
    at ReactJsonSchema.createComponent (bundle.js:49564)
    at ReactJsonSchema.parseSchema (bundle.js:49515)
    at ReactJsonSchema.parseSubSchemas (bundle.js:49535)
    at ReactJsonSchema.parseSchema (bundle.js:49513)
    at ReactJsonSchema.resolveComponentChildren (bundle.js:49588)
    at ReactJsonSchema.createComponent (bundle.js:49564)
    at ReactJsonSchema.parseSchema (bundle.js:49515)
```

I created this pull request which address the issue elsewhere: https://github.com/TechniqueSoftware/react-json-schema/pull/21

Comment (nlaffey) [reply lib/ReactJsonSchema.js]: Yep approved

**Pull request merged**